### ISSUE TITLE
Recently added item not appearing when clicked on

### ISSUE THREAD

**Issue by mello99** [VTUL/vtechworks#330]
Hi there, I clicked on a newly uploaded item in VTW this morning, and when I did this, I was not taken to the item itself, but to a list of related items. It's worth mentioning that this particular item is a duplicate of another item with the same title (http://hdl.handle.net/10919/71780 - no error occurs when I click on this item in "Recently Added"). It appears that these duplicates were generated by DSpace - there aren't really 2 items, in fact - just 2 records for the same item (with the same handle). One of these records just redirects you to a list of "Related Items", then to the original record (if you click on it). Does that make sense?

![vtw_wrong_redirect](https://cloud.githubusercontent.com/assets/5408616/16808185/743026b6-48e9-11e6-9198-95f239fd6a6b.png)

When I click on the link circled in red, I'm then taken to a list of related items:

![vtw_wrong_landing_page](https://cloud.githubusercontent.com/assets/5408616/16808215/99cca1e2-48e9-11e6-9f0a-03d9c2625bd4.png)

Although I can eventually reach the desired item by clicking on it (it's the first item in the list of related items), this is a cumbersome step to have to go through. Thanks, take care.

Melissa

**Comment by pgray64** [comment]
https://vtechworks.lib.vt.edu/handle/10919/71777 shows up as a normal item with two attached PDF's when inspected with the admin item viewer.

I'll look into why it is displaying a recent items listing instead of the normal simple and detailed view.

**Comment by alawvt** [comment]
The collection view with the duplicates shown in the first screenshot is at https://vtechworks.lib.vt.edu/handle/10919/70873.  

The related item list in the second screenshot is at https://vtechworks.lib.vt.edu/handle/10919/71777. This seems like an item with no display of the item metadata and bitstream info. It has Internal ID: 82072.

The correct item with the normal simple item view is at https://vtechworks.lib.vt.edu/handle/10919/71780. It has internal ID: 82075.

**Comment by alawvt** [comment]
Since this duplicate was created by a user in Elements, it would be difficult for us to recreate and troubleshoot. After withdrawing or deleting the duplicate (Internal ID: 82072), I suggest closing this issue and reopening it if this occurs again.

**Comment by keithgee** [comment]
All, I believe that this issue is likely to be corrected by my fix for issue #155 - (Invisible Submission - or something to that effect).
It would be fun to see that, if we're not in a hurry to delete or withdraw the duplicates.

**Comment by alawvt** [comment]
@keithgee, I thought the same thing when I read your comment about the metadata retrieval error caused by the lock symbol. If we do wait to delete this item we could see if the problem is cured by your fix. 

I checked the policies for the good item (internal ID: 82075) and it has one "Anonymous" policy per bitstream. The bad item (Internal ID: 82072) has one "Anonymous" policy plus 4 policies with no group per bitstream. That is probably causing the error. Why this occurred, I have no idea.

**Comment by keithgee** [comment]
That coincides with what I saw in the code, too. Policies that don't have an associated group caused the error with the special lock symbol code.

**Comment by amandafrench** [comment]
Will leave this open for now: let's make sure to test it before the end of the month.

**Comment by keithgee** [comment]
I have just tested this on https://vtechworks.dev.lib.vt.edu/handle/10919/71777, where the Koala changes have been deployed, and the item is appearing as normal, even when not logged in.

The "duplicate" problem mentioned here may be a separate issue from the "not appearing" issue. The user may have submitted twice, or even deleted a file or withdrawn a license, as per the elements configuration:

```

# Workflow Behaviour on File Delete
# ---------------------------------
#
# What should the system do if a delete request is received for
# a file which is already publicly available.
#
# The available options are:
#
#   delete   : delete the file anyway
#   workflow : a new item without the deleted file will be injected into the
#              workflow
#
publications.bitstream.delete.if-published = workflow
```

We can close this, as the problem with displaying items here, and in #155 is fixed, but @mello99 may wish to take care of the duplicate on production as @alawvt previously suggested.

**Comment by amandafrench** [comment]
Yes, that "remove license" option in Elements is very annoying: it really always affects the whole item and not just the license file. I've changed the settings in the Repository Connection on Elements production to the following: 

Allow file deletion when item is in review = true
Allow file deletion when item is live = true

Both those options were false before. I've made myself a note to change the config settings & documentation in all the pertinent places. Let's keep the duplicate on production until we test this fix there.
